models/diffusion_connection.h

```diff
@@ -0,0 +1,189 @@
+/*
+ *  diffusion_connection.h
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+
+/* BeginDocumentation
+Name: diffusion_connection - Synapse type for instantaneous rate connections
+between neurons of type siegert_neuron.
+
+
+Description:
+ diffusion_connection is a connector to create instantaneous connections
+ between neurons of type siegert_neuron. The connection type is identical to
+ type rate_connection for instantaneous rate connections except for the two
+ parameters "drift_factor" and "diffusion_factor" substituting the
+ parameter "weight".
```

Comment:
Thank you for the suggestions. We added a detailed explanation. Note that these factor are quite general and not necessarily in the form of (28) and (29) in the paper.